x/rollapp/keeper/msg_server_create_rollapp.go

```diff
@@ -14,11 +14,27 @@ func (k msgServer) CreateRollapp(goCtx context.Context, msg *types.MsgCreateRoll
 		return nil, types.ErrRollappsDisabled
 	}
 
+	if err := msg.ValidateBasic(); err != nil {
```

Comment:
this redundent as the `msg.ValidateBasic()` already called by the SDK before getting into this function

Reply:
fixed